src/components/subHeader/SubHeader.js

```diff
@@ -86,47 +57,59 @@ class SubHeader extends React.Component {
     );
   };
 
-  render() {
-    const { isLoading, errors, restaurantDetails } = this.state;
-
-    if (isLoading) {
-      return <Spinner text="Loading..." />;
-    }
+  const crumbs = useMemo(() => {
+    return breadCrumbConfig(city, locality, name);
+  }, [city, locality, name]);
 
-    if (errors) {
-      const { message, statusCode } = errors;
-      return <ErrorHandler message={message} statusCode={statusCode} />;
-    }
-
-    const thumbnail = restaurantReader.thumbnail(restaurantDetails);
-    const name = restaurantReader.name(restaurantDetails);
-    const category = restaurantReader.category(restaurantDetails);
-    const street = restaurantReader.street(restaurantDetails);
-    const locality = restaurantReader.locality(restaurantDetails);
-    const city = restaurantReader.city(restaurantDetails);
-    const offers = restaurantReader.offers(restaurantDetails);
+  if (_isEmpty(restaurantDetails)) {
```

Comment:
Take this into restaurant component

Reply:
But it's job is just to render SubHeader and DishItems